lib/Utils/Backends.pm

```diff
@@ -83,4 +84,9 @@ sub is_svirt_except_s390x {
 
 }
 
+sub is_spvm_backend {
```

Comment:
Please rename to is_spvm.

Reply:
You also will need to update the call to is_remote_backend, accordingly.

Reply:
yes